apps/silverback-drupal/web/themes/custom/silverback_iframe_test/silverback_iframe_test.info.yml

```diff
@@ -3,7 +3,7 @@ type: theme
 base theme: silverback_iframe_theme
 description: 'A sub-theme for silverback_iframe_theme.'
 package: Silverback Tests
-core_version_requirement: ^8 || ^9 || ^10
+core_version_requirement: ^8 || ^9 || ^10 || ^11
```

Comment:
Do we really need `8+` and `9+` ?

Also, wouldn't something like `>8 || <12` work to do the same thing?

Reply:
No, yes 🙂 

Keeping 8 and 9 does not hurt. Using `^` is more common in Drupal world.